GoogleSignIn/Sources/GIDVerifyAccountDetail/Implementations/GIDVerifiedAccountDetailResult.m

```diff
@@ -85,16 +87,15 @@ - (void)updateVerifiedDetailsWithTokenResponse:(nullable OIDTokenResponse *)toke
   }
 }
 
-- (nullable NSString *)accessTokenString {
-  return self.verifiedAuthState.lastTokenResponse.accessToken;
-}
-
-- (nullable NSString *)refreshTokenString {
-  return self.verifiedAuthState.refreshToken;
+- (nullable GIDToken *)accessToken {
```

Comment:
This will create a new `GIDToken` every time it's called. I don't think we should do that. Why not just make this a property and create it upon instantiation (and update it upon refresh)?

Reply:
This makes sense, definitely don't want to create a new one every time. 

Though, in regards to refreshing, it looks like `GIDToken.tokenString` and `GIDToken.expirationDate` are [readOnly](https://github.com/google/GoogleSignIn-iOS/blob/f54b8c7aaaf70a2dfc372ac65d05714fa093cf5b/GoogleSignIn/Sources/Public/GoogleSignIn/GIDToken.h#L25-L28), so the current token will not be updated. Seems like we could create a new token on refresh or change the `readonly` to `readwrite`, but I understand that the latter is unlikely. What do you think?

Reply:
We don't want these tokens to be `readwrite`. They should be `readonly`.

Reply:
Sounds good, I moved them to the initializer and replace the access token when refreshing.